examples/disable-personal-account-javascript/src/main.js

```diff
@@ -77,7 +127,24 @@ function escapeHtml(str) {
     .replaceAll(">", "&gt;")
     .replaceAll('"', "&quot;")
     .replaceAll("'", "&#039;");
-};
+}
+
+function showError(errorEl, message) {
+  errorEl.textContent = message;
+  errorEl.style.display = "block";
+}
```

Comment:
I noticed this function is being used for both success/error messages, so we might want to create separate elements for it